app/modules/topics/actionTypes.js

```diff
@@ -49,7 +125,19 @@ export type RemoveAction = {
   },
 };
 
-// TASK SAGA ERRORS
+export type GetAction = {
+  type: typeof GET,
+  payload: {
+    id: Identifier,
+  },
+};
+
+export type GetAllAction = {
```

Comment:
Maybe we should rename this to `GetAllByUserIdAction`? Since you can technically also get all topics, regardless of user...